src/styles/card.css

```
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+  margin-top: 40px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  color: var(--secondary-color);
+  margin-top: 4px;
+}
+
+.card__rating-panel {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+}
+
+.card__review-counter {
+  font-size: 10px;
+  color: var(--main-color);
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price-label {
+  font-size: 12px;
+  color: var(--secondary-color);
+}
+
+.card__price {
+  font-size: 16px;
+  font-weight: 700;
+  color: var(--main-color);
+}
+
+.card__buy-button {
+  display: flex;
+  margin-top: 24px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+  background-color: var(--button-color);
+  text-align: center;
```

```suggestion
  text-align: center;
```
  align-items: center;
  justify-content: center;